browser/model/cdk.js

```diff
@@ -117,6 +123,22 @@ class CDKInstall extends InstallableItem {
     return virtualbox && virtualbox.isConfigured()
       || (hyperv && hyperv.isConfigured() || this.selectedOption !== 'install');
   }
+
+  createHypervSwitch() {
+    let commands = [
+      '$ErrorActionPreference = "Stop"',
+      '$switchName = if ($env:HYPERV_VIRTUAL_SWITCH) {$env:HYPERV_VIRTUAL_SWITCH} else {"minishift-switch"}',
+      'try { Get-VMSwitch -Name $switchName }',
+      'catch { $adapterName;',
+      '[array]$adapters = Get-NetAdapter -Physical | Where-Object { $_.status -eq "up" }',
+      'foreach ($adapter in $adapters) {',
```

Comment:
What if I have more than one adapter?

It seems that we should ether detect adapter with internet connection and use it or change selection page UI to allow some custom settings for installable components.

Reply:
Well, this script searches for all physical adapters that are "up", which means "enabled" and "connected". Not sure if "connected" means the cable is connected, or there is internet connection. 
If some of them are labeled "Ethernet" (the default name for wired connection), it selects the first one it finds. Otherwise it tries to use any adapter it could find with the above conditions.

What would you like to be improved?

Reply:
Thanks for explanation, it sounds good to me.